### PR TITLE
[wrt] Test for auth dialog password

### PR DESCRIPTION
**Pull request by haoxli**
Impacted tests(approved): new 1, update 0, delete 0
Unit test platform: Crosswalk Project for Android 18.46.453.0
Unit test result summary: pass 1, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-5675

### CODE REVIEW

**Comment by Honry** [comment]
Where's the blocked test? A typo?

File: usecase/usecase-wrt-android-tests/samples/WebPassword/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
```
Comment (Honry) [inline usecase/usecase-wrt-android-tests/samples/WebPassword/index.html]: Please add author and e-mail link.

Comment (haoxli) [inline usecase/usecase-wrt-android-tests/samples/WebPassword/index.html]: Move to
\<link rel="author" title="Intel" href="http://www.intel.com">
\<link rel="author" title="Li Hao" href="mailto:haox.li@intel.com">

It's new format.

**Comment by Honry** [comment]
LGTM

**Pull request merged**